archive/2022-11-15-DssSpell/DssSpell.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-FileCopyrightText: © 2020 Dai Foundation <www.daifoundation.org>
```

Comment:
archive date should match with executive date, 2022-11-16 (see spell description)

Reply:
please redo as follow (and remove previous archive folder via `git rm -r archive/2022-11-15-DssSpell`):
`make archive-spell date=2022-11-16`